catroid/src/main/java/org/catrobat/catroid/ui/SettingsActivity.java

```diff
@@ -256,6 +258,23 @@ private void setNXTSensors() {
 		}
 	}
 
+	@SuppressWarnings("deprecation")
+	private void setHintPreferences() {
+		CheckBoxPreference hintCheckBoxPreference = (CheckBoxPreference) findPreference(SETTINGS_SHOW_HINTS);
+
+		if (!BuildConfig.SNACKBAR_HINTS_ENABLED) {
```

Comment:
Thx, I changed that.
According to the feature toggle: The idea was to only enable the feature by default for the release build. Disabling the feature in the develop-buildType has several benefits (like "working" device tests, simplified debugging, ...)